docs/azure-devops-wiki.md

```diff
@@ -31,4 +31,39 @@ There is default table of contents pane to the left of the wiki content. Here, y
 
 _Note that first entry in `.order` file defines wiki's home page._
 
+## Update wiki
+
+While you are pushing changes to GitHub, you won't see the wiki updated if you don't add Azure as a remote. You can push to multiple repositories simultaneously.
+ 
+ 1. First check if Azure is added: `git remote show origin`. If not add it following these steps.
+ 2. Rename your current remote (most likely named origin) to a different name.
+ 3. You can then add the remote for your second remote repository, in this case, Azure. e.g `git remote add azure https://<YOUR_ID>@dev.azure.com/<YOUR_ID>/foam-notes/_git/foam-notes`. You can get it from: Repos->Files->Clone and copy the URL.
+ 4. Now, you need to set up your origin remote to push to both of these. So run: `git config -e` and edit it.
+ 5. Add the `remote origin` section to the bottom of the file with the URLs from each remote repository you'd like to push to. You'll see something like that:
+ ```bash
+ [core]
+  repositoryformatversion = 0
+  filemode = true
+  bare = false
+  logallrefupdates = true
+  ignorecase = true
+  precomposeunicode = false
```

Comment:
Is this code necessary to show this to work? If it's not, should we remove?

Reply:
No, all of this is added and it's needed. Basically the only you are doing is added an extra remote called azure (adding 3 lines of code) and making origin be able to push to two repositories at the same time (adding one line of code).

Reply:
Ah, got it! Should we only show those three lines of code then?

Reply:
I pushed a commit with your suggestions. Thank you.

Reply:
Awesome, thanks for making those so quickly! Just waiting for those to show up and then we can approve!

Reply:
Yes, I'm having some issues to push my changes right now. I hope to solve them soon.

Reply:
Done.